csrankings-s.csv

```diff
@@ -829,6 +830,7 @@ Shan Lu 0001,University of Chicago,http://people.cs.uchicago.edu/~shanlu,9Ee76gw
 Shan Luo 0001,University of Liverpool,https://cgi.csc.liv.ac.uk/~shanluo,ArnkrbwAAAAJ
 Shan Suthaharan,UNC - Greensboro,https://shansuthaharan.com,nY9aMykAAAAJ
 Shan Wang 0001,Renmin University of China,http://info.ruc.edu.cn/academic_professor.php?teacher_id=150,NOSCHOLARPAGE
+Shan Zuo,University of Connecticut,https://engineering.uconn.edu/person/shan-zuo/,_tCX5K4AAAAJ

```

Comment:
ECE, not CS.

Reply:
Dr. Shan Zuo is from ECE of UConn. She is the director of [Distributed Decision-Making and Learning Lab](https://distributed-decision-learning.engr.uconn.edu/). She has several publications in NeurIPS. Can you please include her too?

Reply:
See above.

Reply:
Emery,

I just removed them from the pull request. Can you please verify again?